org.apache.sling.jcr.repoinit.RepositoryInitializer-aem-cs.config prevents repository startup of AEM Cloud Quickstart due to usage of /etc/tags

## Required Information

* [x] AEM Version, including Service Packs, Cumulative Fix Packs, etc: aem-2021.2.4944.20210221T230729Z-210128
* [x] ACS AEM Commons Version: 5.0.0
* [x] Reproducible on Latest? yes/no


### Expected Behavior

AEM Cloud Quickstart (latest version) starts fine with ACS AEM Commons 5.0.0 installed

### Actual Behavior

Startup fails with
```
14.03.2021 13:02:01.886 *INFO* [Apache Sling Repository Startup Thread #1] org.apache.sling.jcr.repoinit.impl.AclUtil Not adding [LocalAccessControlEntry# principal org.apache.jackrabbit.oak.security.user.SystemUserPrincipalImpl:acs-commons-remote-assets-service, privileges: [rep:write, jcr:versionManagement, jcr:read, crx:replicate], isAllow : true] to path /content/dam since an equivalent access control entry already exists
14.03.2021 13:02:01.888 *ERROR* [Apache Sling Repository Startup Thread #1] com.adobe.granite.repository.impl.SlingRepositoryManager Exception in a SlingRepositoryInitializer, SlingRepository service registration aborted
java.lang.RuntimeException: Failed to set ACL (javax.jcr.PathNotFoundException: Cannot set ACL on non-existent path /etc/tags) AclLine ALLOW {paths=[/content/dam, /etc/tags], privileges=[jcr:read, jcr:versionManagement, rep:write, crx:replicate]}
	at org.apache.sling.jcr.repoinit.impl.AclVisitor.setAcl(AclVisitor.java:72) [org.apache.sling.jcr.repoinit:1.1.32]
	at org.apache.sling.jcr.repoinit.impl.AclVisitor.visitSetAclPrincipal(AclVisitor.java:99) [org.apache.sling.jcr.repoinit:1.1.32]
	at org.apache.sling.repoinit.parser.operations.SetAclPrincipals.accept(SetAclPrincipals.java:56) [org.apache.sling.repoinit.parser:1.6.6]
	at org.apache.sling.jcr.repoinit.impl.JcrRepoInitOpsProcessorImpl.apply(JcrRepoInitOpsProcessorImpl.java:56) [org.apache.sling.jcr.repoinit:1.1.32]
	at org.apache.sling.jcr.repoinit.impl.RepositoryInitializerFactory.processRepository(RepositoryInitializerFactory.java:126) [org.apache.sling.jcr.repoinit:1.1.32]
	at org.apache.sling.jcr.base.AbstractSlingRepositoryManager.executeRepositoryInitializers(AbstractSlingRepositoryManager.java:610) [org.apache.sling.jcr.base:3.1.6]
	at org.apache.sling.jcr.base.AbstractSlingRepositoryManager.initializeAndRegisterRepositoryService(AbstractSlingRepositoryManager.java:558) [org.apache.sling.jcr.base:3.1.6]
	at org.apache.sling.jcr.base.AbstractSlingRepositoryManager.access$300(AbstractSlingRepositoryManager.java:92) [org.apache.sling.jcr.base:3.1.6]
	at org.apache.sling.jcr.base.AbstractSlingRepositoryManager$4.run(AbstractSlingRepositoryManager.java:527) [org.apache.sling.jcr.base:3.1.6]
Caused by: javax.jcr.PathNotFoundException: Cannot set ACL on non-existent path /etc/tags
	at org.apache.sling.jcr.repoinit.impl.AclUtil.setAcl(AclUtil.java:121) [org.apache.sling.jcr.repoinit:1.1.32]
	at org.apache.sling.jcr.repoinit.impl.AclVisitor.setAcl(AclVisitor.java:69) [org.apache.sling.jcr.repoinit:1.1.32]
	... 8 common frames omitted
14.03.2021 13:02:01.888 *INFO* [Apache Sling Repository Startup Thread #1] com.adobe.granite.repository.impl.SlingRepositoryManager Stop requested
14.03.2021 13:02:01.888 *INFO* [Apache Sling Repository Startup Thread #1] com.adobe.granite.repository.impl.SlingRepositoryManager stop: Repository still running, forcing shutdown

```

The reason for this behaviour is that the repoinit at https://github.com/Adobe-Consulting-Services/acs-aem-commons/blob/4e173f2a063cf58de5c35aa2f92546bf6e8c31fc/ui.apps/src/main/content/jcr_root/apps/acs-commons/config/org.apache.sling.jcr.repoinit.RepositoryInitializer-aem-cs.config#L22 tries to set ACLs on the non-existing path `/etc/tags`

### Steps to Reproduce

Just install ACS AEM Commons 5.0.0 on a recent AEM Cloud Quickstart server.

### Links

This has been introduced with #2545

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -9,6 +9,9 @@ The format is based on [Keep a Changelog](http://keepachangelog.com)
 <!-- Keep this up to date! After a release, change the tag name to the latest release -->
 [unreleased changes details]: https://github.com/Adobe-Consulting-Services/acs-aem-commons/compare/acs-aem-commons-4.7.2...HEAD
 
+### Fixed
+- #2546 - org.apache.sling.jcr.repoinit.RepositoryInitializer-aem-cs.config prevents repository startup of AEM Cloud Quickstart due to usage of /etc/tags
+
 ## 5.0.0 - 2021-03-13
 
 ### Changed

--- a/ui.apps/src/main/content/jcr_root/apps/acs-commons/config/org.apache.sling.jcr.repoinit.RepositoryInitializer-aem-cs.config
+++ b/ui.apps/src/main/content/jcr_root/apps/acs-commons/config/org.apache.sling.jcr.repoinit.RepositoryInitializer-aem-cs.config
@@ -19,7 +19,7 @@ end
 create service user acs-commons-remote-assets-service with path system/acs-aem-commons
 set ACL for acs-commons-remote-assets-service
     allow jcr:read, jcr:versionManagement, rep:write, crx:replicate on /content/cq:tags
-    allow jcr:read, jcr:versionManagement, rep:write, crx:replicate on /content/dam, /etc/tags
+    allow jcr:read, jcr:versionManagement, rep:write, crx:replicate on /content/dam
     allow jcr:read on /
 
     # If /etc/tags is used, these ACLs be set manually - if RepoInit defines this structure, it supersedes the use of /content/cq:tags

